packages/polkadot/src/polkadot.system.e2e.test.ts

```diff
@@ -8,3 +8,14 @@ const testConfig: RelayTestConfig = {
 }
 
 registerTestTree(systemE2ETests(polkadot, testConfig))
+
+// TODO: Uncomment Post-AHM on Polkadot
+
+// const testConfigForAssetHub: ParaTestConfig = {
```

Comment:
This also needs to changed: `RelayTestConfig`.

I know it's commented out, but still.